onnx_tf/handlers/backend/dynamic_slice.py

```diff
@@ -10,7 +10,7 @@
 class DynamicSlice(BackendHandler):
 
   @classmethod
-  def version_9(cls, node, **kwargs):
+  def version_1(cls, node, **kwargs):
```

Comment:
Decorate with experimental?
https://github.com/onnx/onnx-tensorflow/pull/359

Reply:
OK, I will update the code to use experimental  decorator.